tests/mtouch/RegistrarTest.cs

```diff
@@ -291,6 +291,10 @@ public void MT4134 ()
 					new { Framework = "ARKit", Version = "11.0" },
 					new { Framework = "BusinessChat", Version = "11.3" },
 					new { Framework = "ClassKit", Version = "11.4" },
+					new { Framework = "AuthenticationServices", Version = "12.0" },
+					new { Framework = "CarPlay", Version = "12.0" },
+					new { Framework = "IdentityLookupUI", Version = "12.0" },
+					new { Framework = "NaturalLanguage", Version = "12.0" },
```

Comment:
Is Network not missing?

Reply:
mtouch doesn't complain about the Network namespace (see the list in https://github.com/xamarin/maccore/issues/954), most likely because there is only C bindings in that namespace.

Reply:
:+1: